codegen.ts

```diff
@@ -84,6 +85,7 @@ function resolveDlopenParameter(typeDefs: TypeDef, type: any): string {
   } else {
     return "pointer";
   }
+  // deno-lint-ignore no-unreachable
   throw new TypeError(`Type not supported: ${t}`);
```

Comment:
Maybe I'm not understanding something but typescript seems to be correct in it claiming that this is unreachable. Why is this needed?

Reply:
don't know why its there wasn't going to risk removing it if someone kept it there to add later on

Reply:
You can remove this line... It is unreachable.